pkg/util/pki/parse.go

```diff
@@ -69,25 +68,6 @@ func DecodePrivateKeyBytes(keyBytes []byte) (crypto.Signer, error) {
 	}
 }
 
-// DecodePKCS1PrivateKeyBytes will decode a PEM encoded RSA private key.
-func DecodePKCS1PrivateKeyBytes(keyBytes []byte) (*rsa.PrivateKey, error) {
```

Comment:
This function was not yet deprecated, but just bundles some x509 calls, so can be easily replaced.